docs/examples/io_tests/execute.sh

```diff
@@ -231,8 +231,15 @@ then
   mv "$LOG" log.txt
 fi
 
-#copy core dump messages to log file
-cp shellOut.txt temp.txt
+#truncate if necessary and copy core dump messages to log file
```

Comment:
@AnkshitJain The proposed change can produce a log file of atmost 102 lines in length. We can arrive at that worst case by having a core dump of more than 50 lines and a log of more than 50 lines.
Is it your intention to send a log of 100 lines to the user? If not, you might want to reduce the lengths to 25 each.

Reply:
We also need a bats tests with the right kind of input repository to test for proper handling of the following cases.

1. normal case (we have this test)
2. only compilation logs of more than 50 lines of text 
3. only core dump of more than 50 lines of text
4. both compilation and core dump logs having more than 50 lines of text

Reply:
I'll add the tests for these cases. I propose making a new bats file, named `evaluation_log.bats`. This will contain the different cases, for both unit tests and io-tests.